resources/aws/bucket.go

```diff
@@ -46,47 +44,25 @@ func (b *Bucket) CreateOrFail() error {
 }
 
 func (b *Bucket) Delete() error {
```

Comment:
They correspond to two different types of resources in AWS, with different semantics.

E.g. bucket names must be globally unique, BOs are scoped within a bucket; buckets must be emptied of BOs before they can be deleted. Etc. etc.